CHANGELOG.md

```diff
@@ -5,7 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add the APIs of the following add-ons:
+  - Network version 0.2.0;
```

Comment:
Better not IMO, most of the functionality will not work with 2.11.

Reply:
Removed...